lib/cellect/server/adapters/panoptes.rb

```diff
@@ -58,8 +58,9 @@ def load_data_for(workflow_id)
               .joins(:workflows)
               .where(workflows: {id: workflow_id})
               .joins("LEFT OUTER JOIN subject_workflow_counts ON subject_workflow_counts.subject_id = set_member_subjects.subject_id")
-              .where('subject_workflow_counts.id IS NULL OR subject_workflow_counts.retired_at IS NULL')
+              .where('subject_workflow_counts.retired_at IS NULL')
```

Comment:
Yep.  Good call.  Does `distinct` need to specify that it's distinct on the `subject_id`?